ubirch/crypto.go

```diff
@@ -266,7 +266,7 @@ func (c *CryptoContext) GetCSR(name string, subjectCountry string, subjectOrgani
 		},
 	}
 
-	priv, err := c.getDecodedPrivateKey(name)
+	priv, err := c.getDecodedPrivateKey(uid)
```

Comment:
I understand that `name` can now be different from `uid` and the `privateKey` is linked to the `uid`, rather then the `name`.